lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -96,8 +99,8 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
       canvasWrapper.drawPath(path, _borderPaint);
     }
 
-    final dataSetMaxValue = data.maxEntry.value;
-    final dataSetMinValue = data.minEntry.value;
+    final dataSetMaxValue = data.maxValue ?? data.maxEntry.value;
+    final dataSetMinValue = data.minValue ?? data.minEntry.value;
```

Comment:
I think this will set the first tick value to data.minValue instead of the center of the chart to data.minValue

For example, if minValue = 0, maxValue = 30, tickCount = 2 the center of the chart would be -15 (unlabeled as only the tick lines are labeled), the first tick will be 0 (data.minValue), the second tick would be 15, and the outermost line would be 30 (unlabeled, data.maxValue)

Is this the expected behavior?

Reply:
Huh. No, not really.

But then doesn't this mean the current implementation would also mislabel the center of the chart?

Reply:
I don't think the center of the chart is mislabeled in the current implementation, but it can be misleading because of how it labels things

The best guess I can come up with for the existing functionality is that the center was deliberately made to be less than the min value to give the data points some visual distance from the center. I'm not sure if we should keep it this way for the default, but I'm hesitant to change it as it would change how existing graphs are displayed. We could potentially make the change in a new major version release if @imaNNeoFighT thinks it is the right path to take

Adding max and min values for the chart is great since it would not be a breaking change

As for the outermost line being labeled, this is something I would like to see be added in the future. I think it isn't currently labeled as it might overlap with the chart titles

Reply:
So I guess my implementation doesn't actually break anything after all? Given that the centerpoint is, from your comment, _supposed_ to be smaller than `minValue`.  
Interestingly, I just tested this by adding a zero datapoint and deliberately setting my `data.minValue` to `0`, and it looks like the center is _clamped_ to zero:
![image](https://user-images.githubusercontent.com/16238446/178187512-489d4463-93c2-42a6-88ec-34e34c16fe58.png)

Same thing happens if I _forgo_ setting `data.minValue`, the chart looks exactly the same.

What's your opinion, did I introduce a bug with this PR?

Reply:
In the current implementation, [points and tick lines do not always line up as they should](https://github.com/imaNNeoFighT/fl_chart/issues/1078) so I'm not sure if the chart you are showing is completely accurate. In particular, the broken current implementation assumes that the chart center is always 0 when drawing points, but assumes that the center is (minValue - ((maxValue - minValue) / tickCount)) when drawing tick lines. I think the center point in your chart would actually be drawn at the 0.0 tick line in the [fixed version](https://github.com/imaNNeoFighT/fl_chart/pull/1084), which would mean that the center of the chart would actually be at -6.0

Can you send me the code you used to create that chart so I can check it against my branch that has the fix for the existing mismatched points and tick lines issue?

Reply:
Oh, the data points not lining up on the ticks precisely is not an issue here, I just wanted to illustrate that a data point of zero seems to be clamped to the center.

I did have another issue with scaling, which I discovered on a chart where the data _should_ have lined up, but my next commit fixes that - I neglected to propagate the `data.minValue` to the data point scaling function.